src/backend/models/Text.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require("mongoose")
+const {Schema} = mongoose;
+
+const textSchema = new Schema({
+    _id: String,
+    tekst: String,
```

Comment:
Some of your stuff is in English whilst other is in Estonian. Try not to mix em. Just keep it in English.